app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
+    @comment.save
```

Comment:
Also I have validations for comment :author and :content, but I was having trouble getting a conditional to re-render the job show page if those validations weren't met. I tried render :'job/show', but that didn't work. I realize I need to provide some kind of identifier for which job show to render but I'm unsure of the syntax I need to use. Any advice?, or is it not necessary? @s-espinosa